src/features/string-replaceall.md

```diff
@@ -104,7 +104,7 @@ The important piece of new functionality lies in that first item. `String.protot
 An experimental implementation of `String.prototype.replaceAll` is available in V8 v8.0 behind the `--harmony-string-replaceall` flag.
 
 <feature-support chrome="no https://bugs.chromium.org/p/v8/issues/detail?id=9801"
-                 firefox="no https://bugzilla.mozilla.org/show_bug.cgi?id=1540021"
+                 firefox="yes https://bugzilla.mozilla.org/show_bug.cgi?id=1608168#c8"
```

Comment:
Do we have a version to put here? (even if it's in nightly, I think it's still worthwhile)